src/Columns/ColumnConst.h

```diff
@@ -131,6 +131,12 @@ class ColumnConst final : public COWHelper<IColumn, ColumnConst>
         ++s;
     }
 
+    bool tryInsert(const Field &) override
+    {
+        ++s;
+        return true;
```

Comment:
Shouldn't we check value or at least type of filed?

Reply:
Sure, better to check, we can create temporary column of nested type and try to insert hte field to it. Will add it